### DISCUSSION TITLE
Clean your room

### DISCUSSION THREAD

**Discussion by igorgue** [LazyVim/LazyVim#1586] [General]
I think I'm very good at configuring vim, I got hit by a crazy performance issues recently, everytime I closed nvim or write to a file I'd have to press an additional key to do the action (quit was `<leader>qqq` instead of just 2 q, save would be `:w<cr><cr>`) and then the most annoying thing that started happening yesterday was that neovim will freeze and I couldn't even quit it, felt like a noob then my CPU would start going crazy and my laptop will freeze.

I tried to reproduce the issue with `repro.lua` but no cigar, it was working perfectly there! But at least I figured how to add my own plugins to repro, I was about to do a alias to `nvim -u repro.lua` to *fix* the issue.

Then I thought of just reinstalling the config, doing this:

```bash
# required
mv ~/.config/nvim{,.bak}

# optional but recommended
mv ~/.local/share/nvim{,.bak}
mv ~/.local/state/nvim{,.bak}
mv ~/.cache/nvim{,.bak}
```

And rerun and sync.

And now it works and it's way more snappier and I bet it doesn't freeze but this took me 2 days to figure out...

**Comment by folke** [comment]
Might be related to https://github.com/LazyVim/LazyVim/issues/1581

**Comment by itsjoeoui** [comment]
Hey @igorgue ! Are you using nvim inside of tmux by any change? and has the 'christoomey/vim-tmux-navigator' plugin enabled?

I am facing the exact same issue as you, and I noticed that without tmux everything is nice and snappy (only needs 2 q to quit where as in tmux that becomes 3 q)

Can't seem to figure out why humm...

**Reply by igorgue** [reply]
@itsjoeoui no I don't use tmux. I noticed it was ts and I disabled the thing :+1: to fix all my problems ever, recently I re-enable it and I see slowness again in quitting neovim.

**Reply by itsjoeoui** [reply]
niceee, actually I just upgrade to neovim 0.10 and the problem disappeared! maybe you should give that a try too :)

**Reply by igorgue** [reply]
@itsjoeoui thanks! There was a bug last week on 0.10 that forced me back, I'm using it now though and works like a charm.

**Reply by itsjoeoui** [reply]
Niceee! Though the bug just came back for me after a while on 0.10 haha, no idea why unfortunately :(
